ebos/eclwriter.hh

```diff
@@ -349,14 +351,27 @@ public:
         Inplace inplace;
         {
             OPM_TIMEBLOCK(outputFipLogAndFipresvLog);
-            inplace = eclOutputModule_->outputFipLog(miscSummaryData, regionData, 0,
+            
+            const std::string version = moduleVersionName();
+
+            auto es_ = simulator_.vanguard().eclState();
+            auto sched_ = simulator_.vanguard().schedule();
+
+            boost::posix_time::ptime start_time = boost::posix_time::from_time_t(sched_.getStartTime());
+
+            inplace = eclOutputModule_->calc_inplace(miscSummaryData, regionData, simulator_.gridView().comm());
+
+            if (simulator_.gridView().comm().rank() == 0){
```

Comment:
I think I'd prefer `collectToIORank_.isIORank()` instead of the explicit `rank() == 0` check.  They do the same thing, of course, but the `isIORank()` check is more explicit about the context.